src/components/comment/CommentForm.jsx

```diff
@@ -56,10 +56,10 @@ const CommentForm = (props) => {
                 placeholder={intl.formatMessage({ id: 'comment.form.placeholder' })}
                 required
                 value={commentValue}
-                onChange={onValueChange}
+                onChange={handleValueChange}
                 ref={formInputRef}
-                onKeyPress={autoResizeTextArea}
-                onKeyDown={autoResizeTextArea}
+                onKeyPress={handleTextAreaKeyPress}
```

Comment:
I don't like here that we lost semantics of the function ... "autoResize" was quite nice here to explain, so I propose following alternatives:

1. handleResizableTextArreaCharacterAdd / handleResizableTextArreaKeyPress
2. having autoResizeTextArea(textArea) or similar method called from handleTextAreaKeyPress = (), i.e. part of the processing would be done in the method handleTextAreaKeyPress, but the actual resize action would be handled in the autoResizeTextArea ..